CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
+
+project(sundown-apiary VERSION 1.0 LANGUAGES C)
+
+add_library(sundown-apiary
```

Comment:
Looking at drafter.gyp libsundown target. There is also building the files in `html` and exposing them as headers. I take it these are entirely unused by Snowcrash/Drafter?

Reply:
Looks like it. Drafter/snowcrash builds, links and executes all tests successfully.